xml/System.Runtime.InteropServices/NativeMemory.xml

```diff
@@ -230,11 +317,24 @@
         <Parameter Name="elementSize" Type="System.UIntPtr" />
       </Parameters>
       <Docs>
-        <param name="elementCount">To be added.</param>
-        <param name="elementSize">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="elementCount">The count, in elements, of the block to allocate.</param>
+        <param name="elementSize">The size, in bytes, of each element in the allocation.</param>
+        <summary>Allocates and zeroes a block of memory of the specified size, in elements.</summary>
+        <returns>A pointer to the allocated and zeroed block of memory.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This method allows `elementCount` and/or `elementSize` to be `0` and will return a valid pointer that should not be dereferenced and that should be passed to free to avoid memory leaks.
```

Comment:
```suggestion
This method allows `elementCount` and `elementSize` to be `0`. It returns a valid pointer that should not be dereferenced and that should be passed to free to avoid memory leaks.
```